libs/collectives/examples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+if (HPX_WITH_TESTS AND HPX_WITH_TESTS_EXAMPLES)
+  add_hpx_pseudo_target(tests.examples.collectives)
```

Comment:
Should these also go under `tests.examples.modules`? Don't really care either way.